SideBar.py

```diff
@@ -90,7 +90,73 @@ def copy(self, source, destination):
 			shutil.copy2(source, destination)
 
 		if self.view:
-			self.view.erase_status('ZZZ')
+			self.view.erase_status('SideBarTools:Copy')
 
 	def description(self):
 		return 'Duplicate File…'
+
+
+def temporary_status_message(view, message, key='SideBarTools', duration=5000):
```

Comment:
Is there a link to confirm this? [The official docs] doesn't say anything about messages from plugins disappearing automatically. I simply went for the explicit way. But yeah, if Sublime handles that, this is out; the less code, the better.